src/PyHyperScattering/SST1RSoXSDB.py

```diff
@@ -523,211 +666,331 @@ def subtract_dark(img,pedestal=100,darks=None):
                 retxr = retxr[:-1]
         except IndexError:
             pass
-        
+
         if return_dataset:
-            #retxr = (index,monitors,retxr)
+            # retxr = (index,monitors,retxr)
             monitors.attrs.update(retxr.attrs)
             retxr = monitors.merge(retxr)
-        
+
         if self.use_chunked_loading:
             # dask and multiindexes are like PEO and PPO.  They're kinda the same thing and they don't like each other.
-            retxr = retxr.unstack('system')
-            
-        return retxr
+            retxr = retxr.unstack("system")
 
+        return retxr
 
-    def peekAtMd(self,run):
+    def peekAtMd(self, run):
         return self.loadMd(run)
 
-    def loadMonitors(self,entry,integrate_onto_images=True,n_thinning_iters=5):
-        '''
-        Load the monitor streams for entry.
-        Args:
-           entry (Bluesky document): run to extract monitors from
-           integrate_onto_images (bool, default True): return integral of monitors while shutter was open for images.  if false, returns raw data.
-           n_thinning_iters (int, default 5): how many iterations of binary thinning to use to exclude shutter edges.
-        
-        '''
+    def loadMonitors(
+        self,
+        entry,
+        integrate_onto_images: bool = True,
+        useShutterThinning: bool = True,
+        n_thinning_iters: int = 5,
+    ):
+        """Load the monitor streams for entry.
+
+        Creates a dataset containing all monitor streams (e.g., Mesh Current, Shutter Timing, etc.) as data variables mapped
+        against time. Optionally, all streams can be indexed against the primary measurement time for the images using
+        integrate_onto_images. Whether or not time integration attempts to account for shutter opening/closing is controlled
+        by useShutterThinning. Warning: for exposure times < 0.5 seconds at SST (as of 230209), useShutterThinning=True
```

Comment:
```suggestion
        by useShutterThinning. Warning: for exposure times < 0.5 seconds at SST (as of 9 Feb 2023), useShutterThinning=True
```
this date format is a lot more readable

Reply:
but how will i alphanumeric sort my lines of code based on the datestrings inside them?